arduino/src/arduino.cpp

```diff
@@ -0,0 +1,29 @@
+#include "arduino.h"
+
+Arduino::Arduino(){
+    Arduino::portName = "/dev/ttyUSB0";
+    Arduino::buadRate =9600; 
+    Arduino::timeout = 10000;
+
+}
+
+Arduino::Arduino(std::string fileToWrite,std::string portName, unsigned int buadRate, unsigned int timeout){
+    Arduino::fileToWrite.open(fileToWrite.c_str());
+
+}
+int Arduino::checkConnection(){
+    char helper = Arduino::serial.openDevice(portName.c_str(),buadRate);
```

Comment:
you dont need to write `Arduino::serial`, if you want to be more specific you shuld use `this->serial` but dont use `Arduino::`